snc.sh

```diff
@@ -26,6 +26,7 @@ BASE_DOMAIN=${CRC_BASE_DOMAIN:-testing}
 CRC_PV_DIR="/mnt/pv-data"
 SSH="ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -i id_rsa_crc"
 MIRROR=${MIRROR:-https://mirror.openshift.com/pub/openshift-v4/$ARCH/clients/ocp}
+CERT_ROTATION=${CERT_ROTATION:-true}
```

Comment:
for env vars we are expected to set externally, I prefer something namespaced, "SNC_DISABLE_CERT_ROTATION" for example. `CERT_ROTATION=${SNC_DISABLE_CERT_ROTATION:-enabled}`